docs/fastddsgen/usage/usage.rst

```diff
@@ -81,28 +75,41 @@ Where the option choices are:
        This option expects the location of the template and the location of the file where source code output will be
        stored. |br|
        A custom template example can be found in this `link <https://raw.githubusercontent.com/eProsima/Fast-DDS-Gen/master/resources/Custom.stg>`_
+   * - -flat-output-dir
+     - Ignore input files relative paths and place all generated files in the specified output directory.
+   * - -fusion
+     - Activates fusion.
```

Comment:
We have to talk about this

Reply:
Done in 599a362c6cc12101fed9ddf09b4eaa5796a96cf0